app/src/ui/dispatcher/error-handlers.ts

```diff
@@ -627,6 +628,52 @@ export async function insufficientGitHubRepoPermissions(
   return null
 }
 
+const forkUnreadyErrorMessageRe = /fatal: remote error: access denied or repository not exported: (.+)/s
```

Comment:
This error is a pretty [generic one from Git's dameon process]( https://github.com/git/git/blob/228f53135a4a41a37b6be8e4d6e2b6153db4a8ed/daemon.c#L308-L314) that indicate (in our case) that something went wrong when attempting to read from the repo on disk. Looking at previous support requests at GitHub it appears that this isn't an uncommon error ot see when GitHub file servers are having availability issues for example.

As such I feel uncomfortable having a generic catch-all error handler react so specifically to this generic error.

I think there's a few options. Either we move the detection of a live fork to an in-line fetch loop in the create fork flow itself, or we find a way of determining in this handler whether the repository is recently forked, or we skip handling this all-together until we're certain it's a problem.

Reply:
Thanks @niik! I’m going to close this until we know it’s an issue. We can revisit then with more context about how this presents in desktop.